vision/cloud-client/src/test/java/com/example/vision/DetectIT.java

```diff
@@ -124,7 +124,7 @@ public void testLandmarksGcs() throws Exception {
   @Test
   public void testLandmarksUrl() throws Exception {
     // Act
-    String[] args = {"landmarks", "https://pbs.twimg.com/media/C4wfMOhVcAAUAgq.jpg"};
+    String[] args = {"landmarks", "http://wheresgus.com/landmark.jpg"};
```

Comment:
Good idea, I had tried our existing repos but I'm not sure if any are public. If we don't have a public bucket, maybe it's time for one?